test/unit/RoundsManager.js

```diff
@@ -1,9 +1,9 @@
 import Fixture from "./helpers/Fixture"
 import expectThrow from "../helpers/expectThrow"
```

Comment:
I think the `blockHash()` tests are the only tests that still use this helper. Can we update the `blockHash()` tests to use `expectRevertWithReason` and then remove this import?

Reply:
Ah! Experimented locally and interestingly the revert reason *is* there it just isn't used in `err.reason`:

![Screen Shot 2019-09-23 at 4 20 47 PM](https://user-images.githubusercontent.com/5933273/65459553-2bfe9300-de1e-11e9-9a89-8e73d90dd1e4.png)

This [closed Truffle PR](https://github.com/trufflesuite/truffle/pull/2340) provides some background on the motivation behind not propagating the revert reason into `err.reason` for now.

For this PR, let's just stick with using `expectThrow()` then.

Reply:
Interesting. 

I very much share this thought and it was on my mind during the weekend as well: 

> **Very unclear to me if require should be used in view methods** and last week I opened an issue at EthPM suggesting we amend EIP1319 to define 'null' return values for its Read API because of the lack of clarity around how this stuff is supposed to work. I'm not sure what's normal here.

I'm okay with sticking to `expectThrow` for now although extracting the reason from the error message wouldn't be too hard.

Reply:
Yeah I can see the benefits in moving revert logic out of view functions and just wrapping usage of view functions with a require statement if a state mutating function needs to use the return value of the view functions. But, I think its ok to leave the existing view function structure as-is at this stage since we can still access the revert reason if needed.

I think we should just use `expectThrow` in this PR for now since checking the revert reason for these view functions isn't critical at the moment.

Reply:
I had done a quick check already to see if it would pass when checking the error message rather than the reason field

It does, but can always revert the changes back. 

2eb6a0f

Reply:
Looks like CI tests are failing for a few cases after this change

Reply:
Was a little too fast pushing the commit, fixed now.

Reply:
I had to rebase after cherry-picking but tests should pass now